### PR TITLE
READMES

### PR DESCRIPTION
**Pull request by kariabancroft**
## Description
My first pass at readmes for the first several folders

## Todos
I only got up through HTML/CSS so there may still be some beyond that need to be added

## Feedback Requested
Typos!

### CODE REVIEW

**Comment by droberts-ada** [comment]
Are we keeping the numbers in front of all the markdown files?

File: 00-programming-fundamentals/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Tools
+1. [Introduction to Automated Tests](08-intro-to-automated-tests.md)
+1. [Debugging with Pry](debugging-with-pry.md)
+1. [Code Coverage & SimpleCov](16-code-coverage-and-simplecov.md)
+
+### Git
+1. [Submitting Projects](07-submitting-projects-v1.md)
+1. [Managing Git branches](13-managing-git-branches.md)
+1. [Submit a Project using Branches](15-git-submit-a-project.md)
```
Comment (droberts-ada) [inline 00-programming-fundamentals/README.md]: I don't think we used this `Submit a project using branches` technique for C7, we just had them submit everything against master. Do we want to remove this resource?

File: 00-programming-fundamentals/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+1. [What is Pseudocode?](pseudocode.md)
+
+### Tools
+1. [Introduction to Automated Tests](08-intro-to-automated-tests.md)
+1. [Debugging with Pry](debugging-with-pry.md)
+1. [Code Coverage & SimpleCov](16-code-coverage-and-simplecov.md)
+
+### Git
+1. [Submitting Projects](07-submitting-projects-v1.md)
+1. [Managing Git branches](13-managing-git-branches.md)
```
Comment (droberts-ada) [inline 00-programming-fundamentals/README.md]: I think we put the `Managing git branches` lecture between `diff, pull and merge` and `merge conflicts`

File: 01-ruby-fundamentals/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## Ruby Fundamentals
+
+1. [Welcome to Ruby](01-welcome-to-ruby.md)
+1. [Intro to Object Types](02-intro-to-object-types.md)
+1. [Data Structures](02a-data-structures.md)
+1. [Activity: Divide & Conquer Methods](02b-activity-methods-dc.md)
```
Comment (droberts-ada) [inline 01-ruby-fundamentals/README.md]: Would it make sense to have activities and worksheets as sub-bullets?

File: 05-html-css/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## HTML & CSS
+
+1. [Intro to the Web](01-the-web.md)
```
Comment (droberts-ada) [inline 05-html-css/README.md]: Could we add some more organization here? Maybe something like

### HTML
1. link
1. link

### CSS
1. link
1. link

### Layout
1. link
1. link

File: 05-html-css/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+1. [CSS Selectors](06-css-selectors.md)
+1. [Cascade & Inheritance](09-cascade-inheritance.md)
+1. [Worksheet: Specificity](09a-specificity-practice.md)
+1. [Dev Tools](10-dev-tools-into-default-styles.md)
+
+1. [Box Model](11-box-model.md)
+1. CSS Layout
+    - [Display](12-layout-display.md)
+    - [Floats](13-layout-floats.md)
+    - [Positioning](14-layout-positioning.md)
```
Comment (droberts-ada) [inline 05-html-css/README.md]: There are a bunch of `00-whatever` files in this directory, some of which have similar titles to other lectures. Can we get rid of some?

File: 00-programming-fundamentals/README.md (hunk 1 above)
Comment (kariabancroft) [inline 00-programming-fundamentals/README.md]: I honestly wasn't sure if we used it or not - I still like the idea of them submitting the baseline PR for Rails projects and us merging it in so we don't have to wade through the hundreds of rails boilerplate files....

File: 01-ruby-fundamentals/README.md (hunk 3 above)
Comment (kariabancroft) [inline 01-ruby-fundamentals/README.md]: Yes - I think I started doing this in later Readmes and didn't go back and update

File: 05-html-css/README.md (hunk 4 above)
Comment (kariabancroft) [inline 05-html-css/README.md]: Yes

File: 05-html-css/README.md (hunk 5 above)
Comment (kariabancroft) [inline 05-html-css/README.md]: Yes - but i'm not sure which ones and didn't want to do a deep dive into this as part of this task. I think some of these resources would be useful to use as additional reading. I'll add a Trello card to do some work on this.

**Pull request merged**